app/assets/stylesheets/govuk_publishing_components/components/_tabs.scss

```diff
@@ -31,3 +31,37 @@ li.govuk-tabs__list-item { // stylelint-disable-line selector-no-qualifying-type
     }
   }
 }
+
+// stylelint-disable declaration-no-important
```

Comment:
Can this stylelint rule be applied specifically to the line where `important` is used, rather than the entire block?

Reply:
OK, done. New changelog entry added too.